src/store/migrations.js

```diff
@@ -5,6 +5,22 @@ import { chains, assets as cryptoassets } from '@liquality/cryptoassets'
 import { v4 as uuidv4 } from 'uuid'
 import { shouldApplyRskLegacyDerivation } from './utils'
 
+// add a migration to reuse
+const rskLegacyDerivationMigration = async (state) => {
+  if (!Object.keys(state.accounts)?.length) {
```

Comment:
No need to use `?.length`. also, can you make it more descriptive with variable names?

Reply:
oh ok, I just moved this from the old code, let me check now. thanks!

Reply:
done, fixed @harshjv 

BR!